packages/fxa-auth-server/lib/routes/account.js

```diff
@@ -1195,72 +1227,78 @@ module.exports = (log, db, mailer, Password, config, customs, signinUtils, push,
         const form = request.payload;
         const authPW = form.authPW;
 
-        return customs.check(request, form.email, 'accountDestroy')
```

Comment:
Feeling a bit dangerous here: While adding code to ensure all subscriptions get cancelled along with account deletion, I took a shot at converting the whole method to async / await. I think I covered everything, but can convert it back to plain promises if it seems hinky at all

Reply:
Actually, nevermind: Looks like I did break something in reworking this chunk of code, so I fell back to just adding a new bit in the promise chain to just do what I needed to cancel subscriptions.